### ISSUE TITLE
Default keybind for Flight to change from F to R

### ISSUE THREAD

**Issue by Simuciokas** [Huzuni/Huzuni-1.11#54]
This is so you don't switch to your 2nd hand in the Minecraft versions where such a function is possible

**Comment by brudin** [comment]
Good point! I think R is used with Aura, though that could be moved as well.

**Comment by ili-codes** [comment]
I bind my flight to Minus, it's easy to access with your second hand and isn't in the way.

**Comment by Xer0-Puls3** [comment]
You can change your keybinds easily, I don't think that they should change the default because of your personal preference, It was F in the first place because F is the first letter of flight.

**Comment by Simuciokas** [comment]
It's not for my own personal preference, but because Minecraft already has the F key taken as a default key for switching to 2nd hand which is the main reason for this. Huzuni's versions where the 2nd hand exists to change the default Flight key to a different one - such as the R key. I would also like to mention that having default keys of a cheat client is a bad thing, instead the user should start by configuring the key bindings and settings to his own liking rather than having default presets but this is not the case here
